ray/cluster/base/ray-cluster.yaml

```diff
@@ -0,0 +1,114 @@
+kind: RayCluster
+apiVersion: cluster.ray.io/v1
+metadata:
+  name: 'ray-cluster-example'
+  labels:
+      # allows me to return name of service that Ray operator creates
+      odh-ray-cluster-service: 'ray-cluster-example-ray-head'
+spec:
+  # we can parameterize this when we fix the JH launcher json/jinja bug
+  maxWorkers: 3
+  # The autoscaler will scale up the cluster faster with higher upscaling speed.
+  # E.g., if the task requires adding more nodes then autoscaler will gradually
+  # scale up the cluster in chunks of upscaling_speed*currently_running_nodes.
+  # This number should be > 0.
+  upscalingSpeed: 1.0
+  # If a node is idle for this many minutes, it will be removed.
+  idleTimeoutMinutes: 5
+  # Specify the pod type for the ray head node (as configured below).
+  headPodType: head-node
+  # Specify the allowed pod types for this ray cluster and the resources they provide.
+  podTypes:
+  - name: head-node
+    podConfig:
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        generateName: 'ray-cluster-example-head-'
+      spec:
+        restartPolicy: Never
+        volumes:
+        - name: dshm
+          emptyDir:
+            medium: Memory
+        containers:
+        - name: ray-node
+          imagePullPolicy: Always
+          image: quay.io/thoth-station/ray-ml-worker:v0.2.1
```

Comment:
Ideally we'd have this image in the ODH org. Where's the dockerfile for this image defined?